gui/src/components/JobRuns/Status.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import Icon from '@material-ui/core/Icon'
+
+const styles = theme => {
+  const iconShared = {
+    borderRadius: theme.spacing.unit * 3,
+    padding: 3,
+    width: '1.22em',
+    height: '1.25em'
+  }
+
+  return {
+    waiting: Object.assign(
+      {},
+      iconShared,
+      {backgroundColor: theme.palette.warning.main},
+      {color: theme.palette.warning.contrastText}
+    ),
+    completed: Object.assign(
+      {},
+      iconShared,
+      {backgroundColor: theme.palette.success.light},
+      {color: theme.palette.success.main}
+    ),
+    errored: Object.assign(
+      {},
+      iconShared,
+      {backgroundColor: theme.palette.error.light},
+      {color: theme.palette.error.main}
+    )
+  }
+}
+
+const Status = ({children, classes}) => {
+  if (children === 'completed') {
+    return (
+      <Icon className={classes.completed} title='completed'>
+        done
+      </Icon>
+    )
+  } else if (children === 'errored') {
+    return (
+      <Icon className={classes.errored} title='error'>
+        error_outline
+      </Icon>
+    )
+  }
+
+  return (
+    <Icon className={classes.waiting} title='waiting'>
+      access_time
```

Comment:
I'm assuming these underscored strings match some convention elsewhere, but I can't find it in this PR.

* `access_time`
* `error_outline`
* `done`

It's a shame

* `waiting` != `access_time`
* `completed` != `done`
* `errored` != `error` != `error_outline`

Perhaps there are copy constraints needing a string change but perhaps we can collapse these into one string for consistency? I see the style names above (`errored`, `completed`, `waiting`). Let's try to rename those for consistency with the names in here.

Reply:
@dimroc they conform the [Google Material UI guidelines](https://material.io/tools/icons/?style=baseline).

I'd prefer to keep the class names as-is because that is our domain whereas the icon text is specific to the library we're using.

Reply:
Even with? can we changed errored to error or vice versa?

* errored != error != error_outline

Reply:
I was trying to use the same naming convention as the backend https://github.com/smartcontractkit/chainlink/blob/161e0cfde7ac006162e1f69ce0e85a1df1f6d30e/store/models/common.go.

`waiting` is a grouping of all the unstarted, in progress + pending